src/main/java/net/pms/dlna/DLNAMediaInfo.java

```diff
@@ -2916,11 +2918,11 @@ public Mode3D get3DLayout() {
 			case "oulf":
 			case "top-bottom (left eye first)":
 				isAnaglyph = false;
-				return Mode3D.OUL;
+				return Mode3D.ABL;
 			case "ourf":
```

Comment:
@valib I wasn't aware about Youtube naming convention.
So, to me it looked that as most of the time we use FFmpeg naming convention we should stick to it for a more easy understanding / updating.

That's said, i cannot said that one is more right than the other. But i really don't know where to search for having accurate and updated informations about Youtube naming convention, as we'll need it if they change it. So i hope that i'm the only one in that case.

As i said i'll approve your PR as it is anyways.